doc/adr/0013-synchronous_peer_discovery.md

```diff
@@ -0,0 +1,82 @@
+# 13. Synchronous Peer Discovery
+
+Date: 2023-04-19
+
+## Status
+
+Accepted
+
+## Context
+
+In order to increase security, it is desirable to observe only peers that are on the majority fork.
+This can be determined by querying discovered peers for their latest ordinal values prior to
+selecting one to observe. Afterwards, the majority ordinal can be determined and each peer is queried
```

Comment:
It's not necessarily gonna be the highest number because some nodes may be a little behind while still being in the same fork. I also think outliers may need to be filtered out (some algorithm for deciding which values are too extreme is needed). Actually I guess you would need to find the nodes in the same close range of ordinals and then pick the lowest ordinal from that reasonable range as the ordinal to determine the majority hash. Just food for thought.

Reply:
The majority ordinal is the ordinal value that the highest number of peer nodes reported as their latest ordinal.

Reply:
Created a ticket for this.